src/core/warn.ts

```diff
@@ -2,12 +2,12 @@ import type { ViewStyle } from 'react-native'
 
 export const maybeWarnAboutMultipleUnistyles = (style: ViewStyle, displayName = 'Unknown') => {
     if (__DEV__ && style && !Array.isArray(style)) {
-        const unistylesKeys = Object
-            .keys(style)
-            .filter(key => key.startsWith('unistyles-'))
+        const unistylesKeys = Object.keys(style).filter(key => key.startsWith('unistyles-'))
```

Comment:
what if I prefer my style?

Reply:
I added an example, you can do `// biome-ignore format: I like it better this way` but it's a bit cumbersome to do